client/state/stats/lists/selectors.js

```diff
@@ -77,7 +76,7 @@ export const getSiteStatsPostStreakData = treeSelect(
 		// ensure streakData.data exists and it is not an array
 		if ( streakData && streakData.data && ! isArray( streakData.data ) ) {
 			Object.keys( streakData.data ).forEach( timestamp => {
-				const postDay = i18n.moment.unix( timestamp ).locale( 'en' );
+				const postDay = moment.unix( timestamp ).locale( 'en' );
```

Comment:
We don't need the `locale( 'en' )` call here. The format string `YYYY-MM-DD` doesn't include any i18n-dependent info -- just universal numbers.

Reply:
SGTM! 👍